roles/alertmanager_config/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Install Prometheus Alertmanager.
+  ansible.builtin.import_role:
+    name: cloudalchemy.alertmanager
+  vars:
+    alertmanager_config_file: "/home/{{ _ce_provision.username }}/ce-provision/roles/alertmanager_config/templates/{{ alertmanager_config.alertmanager_config_file }}"
```

Comment:
I would put this path in `defaults/main.yml`, e.g.

```yaml
---
alertmanager_config:
  alertmanager_config_file:  "/home/{{ _ce_provision.username }}/ce-provision/roles/alertmanager_config/templates/{{ alertmanager_config.alertmanager_config_file }}"
```

And then this would be the same as you had it:

```yaml
---
- name: Install Prometheus Alertmanager.
  ansible.builtin.import_role:
    name: cloudalchemy.alertmanager
  vars:
    alertmanager_config_file: "{{ alertmanager_config.alertmanager_config_file }}"
```

And that would be all this role needs! The Galaxy role handles the rest already.

Reply:
Same for the Prometheus role.